pylabrobot/resources/ml_star/mfx_modules.py

```diff
@@ -100,8 +100,8 @@ def MFX_TIP_module(name: str) -> MFXModule:
   )
 
 
-def MFX_DWP_module(name: str) -> MFXModule:
-  """ Hamilton cat. no.: 188042
+def MFX_DWP_rackbased_module(name: str) -> MFXModule:
+  """ Hamilton cat. no.: 188229
```

Comment:
I can't find the 188229 identifier in the document.

Also, I'm not the biggest fan of adopting the rack-vs-container terminology.

Isn't "188042 MULTIFLEX DWP MODULE" a well-defined resource? (page 126 of MICROLAB STAR Line Operator’s Manual)

Reply:
> I can't find the 188229 identifier in the document.

Yes, that is because the document is heavily incomplete.
Interestingly one cannot find the catalogue number either on [Hamilton's website](https://www.hamiltoncompany.com/automated-liquid-handling/carriers) either - it's almost as if Hamilton really doesn't want to sell their products.

But I've got this catalogue number directly from the Hamilton invoice used to purchase this MFX module and cross-verified its correctness using the Labware Assistant 3D:
![image](https://github.com/PyLabRobot/pylabrobot/assets/122165124/68aa9360-24ba-425f-aa10-50775d2731ff)
This is therefore a well-defined Hamilton resource that I have also seen in Synthace videos and other Hamilton setups, including at Hamilton trainings.

> Isn't "188042 MULTIFLEX DWP MODULE" a well-defined resource? (page 126 of MICROLAB STAR Line Operator’s Manual)

It appears that this is actually a "skirted"/"container-based" DWP, i.e. the opposite of what I added to the PLR resource library here:
![image](https://github.com/PyLabRobot/pylabrobot/assets/122165124/c79d7303-f52e-4523-a1bc-eb5e4b5c2372)

> Also, I'm not the biggest fan of adopting the rack-vs-container terminology.

Yes, I agree. A clear definition of the concept is deeply needed for PLR because currently the lack thereof causes crashes (and has to be solved in a hard-coded way using offsets in my experience).
But the word "rack-based" is illogical to mean "sitting on its frame".
If we don't adapt the Hamilton-tested terminology we have to invent our own though, especially to avoid the confusion about naming which causes people to buy the wrong modules and costs up to ~$500 per wrong purchase (in the new-purchase market).
I'm happy to brainstorm ideas of how to incorporate the concept into PLR with different naming and rewrite all labware the takes plates on a specific site.

Reply:
Yes... I do believe the PLR resource library could end up becoming the digital Hamilton labware catalogue that everyone I know wishes exists 😅

Reply:
OK, we should adopt the name from the mfx tool "dwp rack based". That makes sense. Thank you for the detailed reference.